framework/src/source/rooibos/BaseTestSuite.bs

```diff
@@ -2960,16 +2960,6 @@ namespace rooibos
       return instance
     end function
 
-    ' @ignore
```

Comment:
What is this function?
It's never called in the framework, and it requires `mv_getIdsFromStyleJson` and `mc_getArray`?

Reply:
Looks like stuff from Maestro maybe. I wouldn't want to remove it _yet_...

Reply:
This is in v6, so I'm happy to get rid of it :)

Reply:
Yeah I agree. This is the time to do it. Yoink.

Reply:
Eh....the more I think about it, I think maybe we should leave these here. there are several other maestro-specific things in this framework as well, and it IS probably safer to leave things alone so that the brighterscript v1 upgrade for people is as seamless as possible. 

Not saying we can't break anything, but removing assertions is probably not the best thing to do right now.